core/templates/services/html-length.service.spec.ts

```diff
@@ -66,26 +66,47 @@ describe('Html Length Service', () => {
 
   it('should compute length for strings with only paragraph tag', () => {
     const htmlString = '<p>Earth Our home planet is the third planet' +
-      ' from the sun. Despite the vastness of the Solar System and' +
-      ' the rest of space, it is the only place that we know life' +
-      ' exists on. It is at just the right distance from the sun to' +
-      ' ensure a comfortable temperature for us to exist in.</p>';
+      ' from the sun.</p>';
 
     const result = htmlLengthService.computeHtmlLengthInWords(htmlString);
 
-    expect(result).toBe(54);
+    expect(result).toBe(11);
   });
 
   it('should compute length for strings with paragraph tag and' +
-    ' descendants text nodes', () => {
-    const htmlString = '<p><em>This is a brief exploration about' +
-      'conjugations in Spanish.</em>Consider the following sentence' +
-      ':<br><br><em> Yo jugar al fútbol con amigos los domingos.<br>' +
-      '<br></em> What verb conjugation goes in the blank?</p>';
-
-    const result = htmlLengthService.computeHtmlLengthInWords(htmlString);
-
-    expect(result).toBe(26);
+    'descendants text nodes', () => {
+    const testCases = [
```

Comment:
Can you add some test cases with math and image tags?

Reply:
@chris7716, we already have bunch of cases for math and images, you can find them https://github.com/oppia/oppia/pull/18802/files#diff-faa264bf77420ce5737ca9c8d025f37f4a23848af3c92b17aa08111b9255fc7eL91 and in further lines